StationeersLaunchPad/LaunchPadConfig.cs

```diff
@@ -39,18 +42,29 @@ public static class LaunchPadConfig
     public static List<ModInfo> Mods = new();
     public static HashSet<string> GameAssemblies = new();
     public static LoadState LoadState = LoadState.Initializing;
+    public static bool AutoUpdate = false;
     public static bool AutoLoad = true;
+    public static bool HasUpdated = false;
     public static Stopwatch AutoStopwatch = new();
 
     public static async void Run()
     {
+      AutoUpdate = AutoUpdateOnStart.Value;
       AutoLoad = AutoLoadOnStart.Value;
-
+      
       await Load();
 
-      while (LoadState < LoadState.Configuring)
+      while (LoadState < LoadState.Updating)
         await UniTask.Yield();
 
+      if (HasUpdated)
```

Comment:
do we want to force a restart here in order to use mods, or just encourage it? I'm not sure one way or the other, unless it definitely wouldn't work